packages/@adobe/spectrum-css-temp/components/button/skin.css

```diff
@@ -1019,3 +1019,381 @@ governing permissions and limitations under the License.
+  --spectrum-clearbutton-medium-icon-color-key-focus: Highlight;
+  --spectrum-fieldbutton-background-color: ButtonFace;
+  --spectrum-fieldbutton-background-color-disabled: ButtonFace;
+  --spectrum-fieldbutton-background-color-down: ButtonFace;
+  --spectrum-fieldbutton-background-color-hover: ButtonFace;
+  --spectrum-fieldbutton-background-color-key-focus: ButtonFace;
+  --spectrum-fieldbutton-border-color: ButtonText;
+  --spectrum-fieldbutton-border-color-down: Highlight;
+  --spectrum-fieldbutton-border-color-error: ButtonText;
+  --spectrum-fieldbutton-border-color-error-down: Highlight;
+  --spectrum-fieldbutton-border-color-error-hover: Highlight;
+  --spectrum-fieldbutton-border-color-error-key-focus: Highlight;
+  --spectrum-fieldbutton-border-color-hover: Highlight;
+  --spectrum-fieldbutton-border-color-key-focus: Highlight;
+  --spectrum-fieldbutton-icon-color-disabled: GrayText;
+  --spectrum-fieldbutton-placeholder-text-color-key-focus: ButtonText;
+  --spectrum-fieldbutton-quiet-background-color: ButtonFace;
+  --spectrum-fieldbutton-quiet-background-color-disabled: ButtonFace;
+  --spectrum-fieldbutton-quiet-background-color-down: ButtonFace;
+  --spectrum-fieldbutton-quiet-background-color-hover: ButtonFace;
+  --spectrum-fieldbutton-quiet-background-color-key-focus: ButtonFace;
+  --spectrum-fieldbutton-quiet-border-color: ButtonFace;
+  --spectrum-fieldbutton-quiet-border-color-down: Highlight;
+  --spectrum-fieldbutton-quiet-placeholder-text-color-key-focus: ButtonText;
+  --spectrum-fieldbutton-text-color: ButtonText;
+  --spectrum-fieldbutton-text-color-disabled: GrayText;
+  --spectrum-fieldbutton-text-color-hover: ButtonText;
+  --spectrum-fieldbutton-text-color-key-focus: ButtonText;
+  --spectrum-logicbutton-and-background-color: ButtonFace;
+  --spectrum-logicbutton-and-background-color-disabled: ButtonFace;
+  --spectrum-logicbutton-and-background-color-hover: ButtonFace;
+  --spectrum-logicbutton-and-border-color: ButtonText;
+  --spectrum-logicbutton-and-border-color-disabled: GrayText;
+  --spectrum-logicbutton-and-border-color-hover: Highlight;
+  --spectrum-logicbutton-and-text-color: ButtonText;
+  --spectrum-logicbutton-and-text-color-disabled: GrayText;
+  --spectrum-logicbutton-or-background-color: ButtonFace;
+  --spectrum-logicbutton-or-background-color-hover: ButtonFace;
+  --spectrum-logicbutton-or-border-color: ButtonText;
+  --spectrum-logicbutton-or-border-color-hover: Highlight;
+  --spectrum-logicbutton-or-text-color: ButtonText;
+  --spectrum-button-primary-focus-ring-color-key-focus: CanvasText;
+  --spectrum-button-primary-focus-ring-size-key-focus: 3px;
+  }
+  .spectrum-Button--overBackground {
+    &:hover {
+      color: ButtonText;
+    } 
+    &:focus-ring {
+      color: ButtonText;
+    }
+    &.is-active {
+      color: ButtonText;
+    }
+  }
+  .spectrum-ActionButton {
+    &:focus-ring,
+    &.is-focused  {
+        outline: CanvasText solid 3px;
```

Comment:
The focus ring for action buttons seems a bit thick compared to the XD files
Local:
![png2](https://user-images.githubusercontent.com/9661127/146421421-787b283e-a77a-46a9-adab-e9a393fb1fd2.PNG)

XD:
![image](https://user-images.githubusercontent.com/9661127/146421476-852f3b12-d7bc-4a51-b3a0-50c46d10a251.png)

Reply:
Note: Made a change to remove this here: https://github.com/adobe/react-spectrum/commit/b3d5e334dfa5610bb00c5d0917fa73f461c4e825, will need to ask Spectrum if it should be a halo focus ring or not

From XD (emphasized and selected):
![image](https://user-images.githubusercontent.com/9661127/146455269-664be16b-cefd-4be8-b21e-880cf5f00a80.png)

From local:
![fafawfawf](https://user-images.githubusercontent.com/9661127/146455282-4768809c-da95-4af8-bac5-6529ee24ea55.PNG)